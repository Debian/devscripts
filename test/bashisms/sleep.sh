#!/bin/sh

sleep 0.1 # BASHISM
sleep 1s # BASHISM
sleep 1d # BASHISM
sleep -- 1 # BASHISM
sleep 1.5s # BASHISM
sleep 1 2 3 # BASHISM
