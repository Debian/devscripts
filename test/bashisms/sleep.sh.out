possible bashism in bashisms/sleep.sh line 3 (sleep only takes one integer):
sleep 0.1 # BASHISM
possible bashism in bashisms/sleep.sh line 4 (sleep only takes one integer):
sleep 1s # BASHISM
possible bashism in bashisms/sleep.sh line 5 (sleep only takes one integer):
sleep 1d # BASHISM
possible bashism in bashisms/sleep.sh line 6 (sleep only takes one integer):
sleep -- 1 # BASHISM
possible bashism in bashisms/sleep.sh line 7 (sleep only takes one integer):
sleep 1.5s # BASHISM
possible bashism in bashisms/sleep.sh line 8 (sleep only takes one integer):
sleep 1 2 3 # BASHISM
